clients/lightrider/lightrider-ui-features.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global ReportUIFeatures */
+
+/**
+ * Extends ReportUIFeatures to get html from a browserified ReportGenerator.
+ */
+class LightriderUIFeatures extends ReportUIFeatures {
+  /**
+   * Uses ReportGenerator to create the html that recreates this report.
+   * @return {string}
+   * @override
+   */
+  getReportHtml() {
+    // @ts-ignore This is only called in browser in Lightrider where this exists.
```

Comment:
```suggestion
    // @ts-ignore ReportGenerator is put on window by the host page.
```
what about this. The LR part isn't really important the ignore